bundles/org.openhab.core.automation.module.script.rulesupport/src/test/java/org/openhab/core/automation/module/script/rulesupport/internal/loader/DelegatingScheduledExecutorService.java

```diff
@@ -0,0 +1,133 @@
+/**
+ * Copyright (c) 2010-2021 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.core.automation.module.script.rulesupport.internal.loader;
+
+import java.util.Collection;
+import java.util.List;
+import java.util.concurrent.*;
+
+import org.jetbrains.annotations.NotNull;
+
+/**
+ * Implementation of {@link ScheduledExecutorService} which simply delegates to the instance supplied.
+ *
+ * @author Jonathan Gilbert - initial contribution
+ */
+public class DelegatingScheduledExecutorService implements ScheduledExecutorService {
+    private ScheduledExecutorService delegate;
+
+    public DelegatingScheduledExecutorService(ScheduledExecutorService delegate) {
+        this.delegate = delegate;
+    }
+
+    @NotNull
+    @Override
+    public ScheduledFuture<?> schedule(@NotNull Runnable command, long delay, @NotNull TimeUnit unit) {
+        return delegate.schedule(command, delay, unit);
+    }
+
+    @NotNull
+    @Override
+    public <V> ScheduledFuture<V> schedule(@NotNull Callable<V> callable, long delay, @NotNull TimeUnit unit) {
```

Comment:
All the jetbrains `@NotNull` annotations in this file should be replaced with Eclipse JDT annotaitons.